src/core/internal/dassert.d

```diff
@@ -319,9 +331,9 @@ private string miniFormat(V)(const scope ref V v)
 import core.atomic : atomicLoad;
 
 // Inverts a comparison token for use in _d_assert_fail
-private string invertCompToken(string comp)
+private string invertCompToken(string comp) pure nothrow @nogc @safe
 {
-    switch (comp)
+    final switch (comp)
```

Comment:
I don't see how it can be `final` though ?

Reply:
This was initially done to avoid the `@gc` concatenation in the `assert` while still providing a better error.
Now using `combine`  to format the error message.